src/system/obj/Dir.h

```diff
@@ -142,6 +142,9 @@ template <class T1> BinStream& operator>>(BinStream& bs, ObjDirPtr<T1>& ptr){
     return bs;
 }
 
+/** "An ObjectDir keeps track of a set of Objects.  \n
```

Comment:
For consistency, you should write your own brief, then have the description from _objects.dta under an "Original _objects description: " header. For example, 
```
* @brief Contains a set of objects to write to disk. 
* Original _objects description:
* [...]